### DISCUSSION TITLE
🙌🏽 You're a legend!

### DISCUSSION THREAD

**Discussion by srizvi** [nuotsu/sanitypress-2023#26] [General]
I hope you don't mind me sharing this, but your repo has been a great source of information and insights for me getting (back) up to speed on the latest and greatest Sanity stuff. Really appreciate it. Thank you! 🙏🏽 🏆 🐐

**Comment by nuotsu** [comment]
Thanks @srizvi! I appreciate the kind words and glad this project has helped you in your journey. Made my day!

Would it be OK if I use your comment as a testimonial on the [SanityPress](https://sanitypress.dev) homepage?

**Reply by srizvi** [reply]
> Thanks @srizvi! I appreciate the kind words and glad this project has helped you in your journey. Made my day!
> 
> 
> 
> Would it be OK if I use your comment as a testimonial on the [SanityPress](https://sanitypress.dev) homepage?

For sure!

**Reply by nuotsu** [reply]
Thanks again!
